CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changes
 4.4.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add support for general-purpose Python 3.x annotations / type-hints in interface definitions.
```

Comment:
@jamadden my current use case is introducing interfaces into a project with a 6-year history, which includes convincing the team that it has value and doesn't have extra magic and corner/edge cases. Therefore, being able to specify type hints in interface attributes is a good thing to have, even though it might not be correctly handled by mypy at the moment (I haven't run it against interfaces yet, though, I assume it won't infer the type correctly because there are no stubs for `zope.interfaces`). At least, it is good to be able to specify them as in regular python objects.